test/helpers.test.ts

```diff
@@ -0,0 +1,51 @@
+import { isDestroyed, isNew, isTouched } from "../src/helpers";
```

Comment:
Update the test case to reflect its use cases. There are some failing ones, could you try fixing those?